static/js/ui_report.js

```diff
@@ -6,18 +6,13 @@ var exports = {};
    response- response that we want to display
    status_box- element being used to display the response
    cls- class that we want to add/remove to/from the status_box
-   type- used to define more complex logic for special cases
 */
 
-exports.message = function (response, status_box, cls, type, remove_after) {
+exports.message = function (response, status_box, cls, remove_after) {
     if (cls === undefined) {
         cls = 'alert';
     }
 
-    if (type === undefined) {
-        type = ' ';
-    }
-
```

Comment:
Why was it redundant?  Or was it just no longer used?

Reply:
(This probably belongs in the commit message)

Reply:
It's redundant because it's no longer used as I noticed in `git grep` and `git blame`.

Reply:
Changed the commit description.